validation/fault_game_params_test.go

```diff
@@ -0,0 +1,98 @@
+package validation
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-program/prestates"
```

Comment:
> op-program release hash maps to version `1.3.0-rc.3`. Currently the test fails if the hash is not mapped to a standard, non `-rc` tag. What do? Should we change the test to allow `-rc` tags or does op-sepolia need to be updated to use a different op-program prestate?

I recall the prestate @ajsutton originally suggested we used was from a release candidate and not an official release, but I'm unsure why. Do op-program releases need governance approval?

Reply:
op-program defines the derivation process that fault proofs uses and is committed to in the contract upgrade signed by the security council. So the op-program being released is generally called out in the governance proposal. We haven't actually been very consistent with proposing an RC and then finalizing on approval or just proposing the final release (Fjord finalized the 1.2.0 release prior to gov approval). 

op-program _also_ includes the chain configuration so testnets will almost always run RCs not final releases because they get hard forks first without gov approval so need a new op-program version that has the hard fork enabled.  Then mainnet needs a different op-program again since it needs to add in the hard fork activation on mainnet. I think the key thing here is that mainnets need gov approval, testnets do not and hard forks are routinely activated on testnets without gov approval, so testnets should be allowed to use RCs.

The other interesting corner case is for new mainnet chains - they will need a new op-program. It should have the same derivation rules applied, but will need a new op-program release to update the superchain-registry to include the new chain. That version won't be governance approved, I'm not sure if it should be an RC or final release.

Reply:
> I think the key thing here is that mainnets need gov approval, testnets do not and hard forks are routinely activated on testnets without gov approval, so testnets should be allowed to use RCs.

This all makes sense to me. I will implement the following:
* testnets can use any release (full, -rc, etc.) and still pass the validation checks
* mainnets must use a full release with the `governanceApproved` flag set

Reply:
Done here and tidied up in the commits after: https://github.com/ethereum-optimism/superchain-registry/pull/636/commits/26c954e57150857b47866860262cd78b9137acb9